### PR TITLE
Arachnophobia mode

### PR DESCRIPTION
**Pull request by Deuchnord**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none
| License       | WTFPL
| Doc PR        | api-platform/doc#612

One of the greatest bugs in the Human software is their fear of spiders. In some cases, the simple seeing of something big go down the screen makes them feel bad for some seconds, before realizing it's just Webby.

Problem is, the Human software is closed-source, so this PR tries to manage that issue by providing a way to disable displaying Webby on the Swagger UI pages. Of course, Webby remains visible by default.

To disable Webby's display, just add this to API Platform's config:
```yaml
# /config/api_platform.yaml
api_platform:
    show_webby: false # defaults to true
```

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-        web.classList.replace('calm', 'frighten');
-        setTimeout(() => {
-            webby.classList.replace('frighten', 'calm');
-            web.classList.replace('frighten', 'calm');
-        }, 10000);
-    });
+    const webbyClass = document.getElementsByClassName('webby');
+
+    // First, check if Webby is present
+    if(webbyClass.length > 0) {
```
Comment (alanpoulain) [inline src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js]: Prefer an early return in a function instead.

File: src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js [hunk 2]
```diff
@@ -145,3 +131,28 @@ window.onload = () => {
         }
     }
 };
+
+function manageWebbyDisplay() {
+    const webbyClass = document.getElementsByClassName('webby');
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js]: Can be simplified:

```js
const webby = document.getElementsByClassName('webby')[0];
if (!webby) return;
```

File: src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js [hunk 3]
```diff
@@ -133,14 +133,9 @@ window.onload = () => {
 };
 
 function manageWebbyDisplay() {
-    const webbyClass = document.getElementsByClassName('webby');
+    const webby = document.getElementsByClassName('webby')[0];
+    if(!webby) return;
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Resources/public/init-swagger-ui.js]: missing space between `if` and `(`

**Comment by alanpoulain** [comment]
Can't we merge this one?

**Comment by dunglas** [comment]
Thanks @Deuchnord! First PR, but not the last!

**Comment by jeremyFreeAgent** [comment]
Thank you very much!!! <3

**Pull request merged**